bucket/robo3t.json

```diff
@@ -16,11 +16,15 @@
             "Robo 3T"
         ]
     ],
+    "checkver": {
+        "github": "https://github.com/Studio3T/robomongo",
+        "regex": "robo3t-(?<version>[\\d.]+)-windows-x86_64-(?<hash>[0-9a-f]+)"
+    },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://download.studio3t.com/robomongo/windows/robo3t-$version-windows-x86_64-$matchRandom.zip",
-                "extract_dir": "robo3t-$version-windows-x86_64-$matchRandom"
+                "url": "https://download.studio3t.com/robomongo/windows/robo3t-$version-windows-x86_64-$matchHash.zip",
```

Comment:
Please change download links to GitHub as well

Reply:
@pratikpc Done.